bundler/spec/fixtures/projects/bundler1/gemspec_with_require_no_lockfile/example.gemspec

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require "find"
```

Comment:
As discussed synchronously, the use of `Find` in these gemspecs is atypical and not suggested by Bundler v1.17.x or v2.x.x when generating new gems.

Since `Find` isn't in the ruby context by default, we'd expect `.gemspecs` in the real world to have this require